download_deps.sh

```diff
@@ -1,3 +1,15 @@
+if ! command -v curl &> /dev/null
```

Comment:
&> is a bashism, so you should either remove it (just > works fine) or add a shebang
```suggestion
if ! command -v curl > /dev/null
```

Reply:
ty